mtgjson5/providers/tcgplayer.py

```diff
@@ -276,11 +459,53 @@ def generate_mtgjson_sealed_product_objects(
 
         mtgjson_sealed_products = []
 
+        with constants.RESOURCE_PATH.joinpath("booster_box_size_overrides.json").open(
+            encoding="utf-8"
+        ) as f:
+            booster_box_size_overrides = json.load(f)
```

Comment:
might not be a good idea to call it booster box, since this is every type of product now?

Reply:
ok!